_docs/howto/howto-gcp-deployment.md

```diff
@@ -0,0 +1,102 @@
+# Deploying Forseti on Google Cloud Platform
+
+This page explains how to use the Google Cloud Deployment Manager (DM) to set
+up Forseti for your Google Cloud Platform (GCP) resources. The DM templates
+help you manage configuration for Forseti Inventory and Scanner. It doesn't
+currently schedule or execute Enforcer. You'll use DM to do the following:
+
+  - Create a Cloud SQL instance and database to store Inventory data.
+  - Create a Cloud Storage bucket for Scanner output.
+  - Create a Google Compute Engine instance to deploy and run Forseti Security.
+  - Manage Forseti Security configuration and automatically run Forseti
+  Inventory and Scanner.
+
+## Before you begin
+
+To complete this quickstart, you will need:
+
+  - A GCP project for Forseti Security with billing enabled.
+  - A GCP organization.
+
+## Setting up Forseti Security
+
+{% capture include_content %}
+{% include howto_prereqs_include.md %}
```

Comment:
A normal `include` of Markdown files within a Markdown file suffices without the need to `capture` and output.